README.md

```diff
@@ -22,210 +23,139 @@
-
-## Features
+Verify that the kmcp CLI is installed.
 
-The `kmcp` CLI provides a set of commands to manage the entire lifecycle of your MCP server:
-
-| Command | Description |
-|---------|-------------|
-| `kmcp init` | Scaffolds a new MCP server project with FastMCP (Python) or MCP Go SDK |
-| `kmcp add-tool` | Adds a new tool to your project with automatic boilerplate and registration |
-| `kmcp run` | Runs the MCP server locally for development and testing |
-| `kmcp build` | Builds a Docker image for your MCP server |
-| `kmcp install` | Installs the `kmcp` controller and CRDs on a Kubernetes cluster (one-time setup) |
-| `kmcp deploy` | Deploys your MCP server to Kubernetes behind a pre-configured Transport Adapter |
-| `kmcp secrets` | Manages secrets for your MCP server deployment in Kubernetes |
-
-
-## Architecture
-
-The following diagram illustrates the `kmcp` workflow, from local development to a production deployment in Kubernetes:
-
-```mermaid
-graph TD
-    subgraph Local Development
-        A[Developer] -- kmcp init --> B(MCP Project);
-        B -- kmcp add-tool --> B;
-        B -- kmcp run --> C{Local MCP Server};
-        A -- Edits Code --> B;
-    end
-
-    subgraph Production Deployment
-        B -- kmcp build --> D[Docker Image];
-        D -- kmcp deploy --> E(Kubernetes Cluster);
-    end
+```bash
+kmcp --help
+```
 
-    subgraph Kubernetes Cluster
-        F[kmcp Controller] -- Manages --> G(MCP Server CRD);
-        G -- Deploys --> H[Transport Adapter];
-        H -- Proxies Traffic --> I[MCP Server Pod];
-    end
+<img src="img/cli-help-nov-25.png" alt="kmcp cli help text" width="800">
 
-    A -- Interacts with --> C;
-    E -- Contains --> F;
-    E -- Contains --> G;
-    E -- Contains --> H;
-    E -- Contains --> I;
-```
+_You're ready to go! Continue on to [From Prototype to Production](#from-prototype-to-production) or explore our docs at [kagent.dev/docs/kmcp](https://kagent.dev/docs/kmcp)_
 
-### How It Works
+## From Prototype to Production
 
-1. **Local Development**: Developers use `kmcp init` to scaffold a new project and `kmcp add-tool` to add tools. The server runs locally with `kmcp run` for rapid iteration.
+<img src="img/kmcp.png" alt="kmcp architecture" width="800">
 
-2. **Build Phase**: `kmcp build` packages your MCP server into a Docker image, handling all dependencies and configuration automatically.
+MCP connectivity affects everyone in an organization. Below are some common quickstart journeys that you may experience:
 
-3. **Deployment**: `kmcp deploy` creates a Kubernetes custom resource that the controller watches. The controller then:
-   - Deploys your MCP server as a pod
-   - Configures and deploys a Transport Adapter as a sidecar or separate service
-   - Sets up networking and ingress rules
-   - Manages secrets and environment variables
+- [Your first MCP service prototype](/devel/quickstart/first-prototype.md)
```

Comment:
would these be better in the docs?

Reply:
I updated to point to the existing docs. I think there are more quickstarts we could add, but it won't be done as part of this PR.